de/firefox/new/sem.lang

```diff
@@ -0,0 +1,57 @@
+## NOTE: demo server available at https://www-dev.allizom.org/firefox/new/?xv=fast
+
+
+# HTML page title
+;Free Web Browser for Android, iOS, Desktop
+Free Web Browser for Android, iOS, Desktop
+
+
+# HTML page description
+;Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+
+
+;More Browser Speed
+More Browser Speed
+
+
+;Bring it, Internet! We’ve spent the last year supercharging Firefox’s performance. Now start up faster, tab hop quicker, and scroll like a speed demon.
+Bring it, Internet! We’ve spent the last year supercharging Firefox’s performance. Now start up faster, tab hop quicker, and scroll like a speed demon.
+
+
+;More Browser Privacy
+More Browser Privacy
+
+
+;Firefox doesn’t sell access to your personal information like other companies. From privacy tools to tracking protection, you’re in charge of who sees what.
+Firefox doesn’t sell access to your personal information like other companies. From privacy tools to tracking protection, you’re in charge of who sees what.
+
+
+;More Browser Freedom
+More Browser Freedom
+
+
+;Following the pack isn’t our style. As part of the non-profit Mozilla, Firefox leads the fight to protect your online rights, and champion an Internet that benefits everyone—not just a few.
+Following the pack isn’t our style. As part of the non-profit Mozilla, Firefox leads the fight to protect your online rights, and champion an Internet that benefits everyone—not just a few.
+
+
+;Browse freely with Firefox today.
+Browse freely with Firefox today.
+
+
+;The fastest Firefox ever
+The fastest Firefox ever
+
+
+;The new multi-process Firefox is faster than ever and uses less memory than Chrome, so you can browse smoothly and leave plenty of memory to keep your computer snappy.
+The new multi-process Firefox is faster than ever and uses less memory than Chrome, so you can browse smoothly and leave plenty of memory to keep your computer snappy.
+
+
+;Thanks for choosing Firefox
+Thanks for choosing Firefox
+
+
+;Your download should begin automatically. <br>If not, <a id="%(id)s" href="%(fallback_url)s">click here</a>.
+Your download should begin automatically. <br>If not, <a id="%(id)s" href="%(fallback_url)s">click here</a>.
+
+
```

Comment:
Extra empty lines

Reply:
All localized files have two empty lines at the bottom, they're added by lang_update